### DISCUSSION TITLE
I don't know why I didn't get a GitHub Education

### DISCUSSION THREAD

**Discussion by sungwokawk** [community/community#138230] [GitHub Education]
### Hi everyone,

Hi everyone,
Hi, I'm currently enrolled in a university in the Republic of Korea, and would like to apply for the Student Developer Pack.

However, my applications kept being rejected, and the reason for the rejection simply said "Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Sep 10, 2024" Which does not make much sense since my photos were successfully uploaded and processed.

How can I have my documents reviewed by a real person, or how can I have my documents successfully recognized? Any help would be greatly appreciated.

Seong woo Kawk

**Comment by Lokeshsuwalka05** [comment]
The same thing happened to me. However, GitHub provided some instructions on why my application was rejected:
1)Please secure your GitHub account with two-factor authentication.
2)Please complete your GitHub billing information.
Complete these steps, and after that, you can reapply.
After completing these steps, my pack was successfully approved..

![rough](https://github.com/user-attachments/assets/2ecba3ec-6d3d-4f76-80d8-9997ec1c89f4)
